src/main/java/entity/FileTranslator.java

```diff
@@ -0,0 +1,92 @@
+package entity;
+
+import java.io.File;
+
+import data_access.Constants;
+import data_access.DBTranslateFileDataAccessObject;
+import use_case.translateFile.DataAccessException;
+import use_case.translateFile.TranslateFileDataAccessInterface;
+
+/**
+ * The representation of a File Translator.
+ */
+public class FileTranslator extends TextTranslator {
+
+    private final TranslateFileDataAccessInterface dataAccessObject = new DBTranslateFileDataAccessObject();
```

Comment:
The FileTranslator is an entity, so it should not depend on the DAO. You should use dependency injection here to pass it through FileTranslator's constructor.